client/src/App.js

```diff
@@ -8,6 +9,14 @@ import { theme } from "./themes/theme";
 import Routes from "./routes";
 
 function App() {
+  useEffect(() => {
+    const getCsrfToken = async () => {
+      const { data } = await axios.get("/csrf-token");
+      axios.defaults.headers.post["X-CSRF-Token"] = data.csrfToken;
```

Comment:
You need to add `delete` to be able to log out!

Reply:
Thanks for catching that! I added delete to logout in thunk creators